common-lib/src/main/scala/com/gu/mediaservice/lib/aws/S3.scala

```diff
@@ -29,10 +30,24 @@ class S3(credentials: AWSCredentials) {
   lazy val client: AmazonS3 =
     new AmazonS3Client(credentials) <| (_ setEndpoint s3Endpoint)
 
-  def signUrl(bucket: Bucket, url: URI, expiration: DateTime): String = {
+  private def removeExtension(filename: String): String = {
+    val regex = """\.[a-zA-Z]{3,4}$""".r
+    regex.replaceAllIn(filename, "")
+  }
+
+  def signUrl(bucket: Bucket, url: URI, image: Image, expiration: DateTime): String = {
     // get path and remove leading `/`
     val key: Key = url.getPath.drop(1)
-    val request = new GeneratePresignedUrlRequest(bucket, key).withExpiration(expiration.toDate)
+
+    val filename: String = image.uploadInfo.filename match {
+      case Some(f)  => s"${removeExtension(f)} (${image.id}).jpg"
+      case _        => s"${image.id}.jpg"
+    }
+
+    val headers = new ResponseHeaderOverrides()
+    headers.setContentDisposition("attachment; filename=\"" + filename + "\"")
```

Comment:
Note that you can use string interpolation with `s"""attachnment; filename="$filename""""`.